<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_add_flag"/>
   <addaction name="separator"/>
   <addaction name="act_edit_flag"/>
   <addaction name="separator"/>
   <addaction name="act_delete_flag"/>
   <addaction name="separator"/>
   <addaction name="act_exit"/>
   <addaction name="separator"/>
  </widget>
  <action name="act_add_flag">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/gnome/22x22/actions/add.png</normaloff>../../../../../usr/share/icons/gnome/22x22/actions/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить новый флаг</string>
   </property>
  </action>
  <action name="act_edit_flag">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/gnome/22x22/apps/accessories-text-editor.png</normaloff>../../../../../usr/share/icons/gnome/22x22/apps/accessories-text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать флаг</string>
   </property>
  </action>
  <action name="act_delete_flag">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/gnome/22x22/actions/remove.png</normaloff>../../../../../usr/share/icons/gnome/22x22/actions/remove.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить флаг</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../usr/share/icons/gnome/22x22/actions/gtk-quit.png</normaloff>../../../../../usr/share/icons/gnome/22x22/actions/gtk-quit.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="toolTip">
    <string>Выйти из программы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
